core-service/src/test/java/com/crmfoodestablishment/coreservice/web/OrderControllerTest.java

```diff
@@ -0,0 +1,43 @@
+package com.crmfoodestablishment.coreservice.web;
+
+import com.crmfoodestablishment.coreservice.dto.order.NewOrderDto;
+import com.crmfoodestablishment.coreservice.service.OrderService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class OrderControllerTest {
```

Comment:
Controllers can't be tested in this way
You need to use mockMvc for that

Reply:
used mockMvc for that.